components/more_channels/index.js

```diff
@@ -35,6 +37,7 @@ function mapStateToProps(state) {
         teamId: team.id,
         teamName: team.name,
         channelsRequestStarted: state.requests.channels.getChannels.status === RequestStatus.STARTED,
+        canShowArchivedChannels: isMinimumServerVersion(state.entities.general.serverVersion, 5, 18),
```

Comment:
I am guessing you copied this because you needed this for the mobile app? It's not required for the webapp since the latest webapp version is shipped with the server.

Reply:
Ah that makes sense! I did copy it over because of that - I'll remove it again. Thanks!